CHANGELOG.md

```diff
@@ -1,5 +1,144 @@
+
+* [--browser-init-timeout](./docs/articles/documentation/reference/command-line-interface.md#--browser-init-timeout-ms) CLI option
+
+```sh
+testcafe chrome my-tests --browser-init-timeout 180000
+```
+
+* [browserInitTimeout](./docs/articles/documentation/reference/configuration-file.md#browserinittimeout) configuration option
+
+```json
+{
+    "browserInitTimeout": 180000
+}
+```
+
+* [runner.run Method](./docs/articles/documentation/reference/testcafe-api/runner/run.md) parameter
+
+```js
+runner.run({ "browserInitTimeout": 180000 })
+```
+
+This setting sets an equal timeout for local and [remote browsers](./docs/articles/documentation/guides/concepts/browsers.md#browsers-on-remote-devices).
+
+#### Improved `Unable To Establish Browser Connection` Error Message ([PR #5720](https://github.com/DevExpress/testcafe/pull/5720))
+
+TestCafe raises this error when at least one local or remote browser was not able to connect. The error message now includes the number of browsers that have not established a connection.
+
+TestCafe raises a warning if low system performance is causing the connectivity issue.
+
+#### :gear: An Option to Retry Requests for the Test Page ([PR #5738](https://github.com/DevExpress/testcafe/pull/5738))
+
+If a tested webpage was not served after the first request, TestCafe can now retry the request.
+
+You can enable this functionality with a command line, API, or configuration file option:
+
+* the [--retry-test-pages](https://devexpress.github.io/testcafe/documentation/using-testcafe/command-line-interface.html#--retry-test-pages) command line argument
+
+    ```sh
+    testcafe chrome test.js --retry-test-pages
+    ```
+
+* the [runner.run](https://devexpress.github.io/testcafe/documentation/using-testcafe/programming-interface/runner.html) option
+
+    ```js
+    runner.run({
+        retryTestPages: true
+    });
+    ```
+
+* the [retryTestPages](https://devexpress.github.io/testcafe/documentation/using-testcafe/configuration-file.html#retrytestpages) configuration file property
+
+    ```json
+    {
+        "retryTestPages": true
+    }
+    ```
+
+### Bug Fixes
+
+* Fixed a bug where `Selector.withText` couldn't locate elements inside an `iframe` ([#5886](https://github.com/DevExpress/testcafe/issues/5886))
+* Fixed a bug where TestCafe was sometimes unable to detect when a browser instance closes ([#5857](https://github.com/DevExpress/testcafe/issues/5857))
+* You can now install TestCafe with `Yarn 2` ([PR #5872](https://github.com/DevExpress/testcafe/pull/5872) by [@NiavlysB](https://github.com/NiavlysB))
+* Fixed a bug where the `typeText` action would sometimes not be able to replace existing text ([PR #5942](https://github.com/DevExpress/testcafe/pull/5942) by [@rueyaa332266](https://github.com/rueyaa332266))
```

Comment:
@albertov05  A new entry in this PR since your last visit, please take a look

Reply:
I'd like to simplify "would sometimes not be able to replace", if possible. What do you think about the following?

`Fixed a bug where the `typeText` action does not always replace existing text`

Reply:
awesome

Reply:
Thank you.